src/ZoneServer/Buffs/Handlers/Agility_Buff.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Melia.Shared.Game.Const;
+using Melia.Zone.Buffs.Base;
+using Melia.Zone.Network;
+using Melia.Zone.World.Actors.Characters;
+using Melia.Zone.World.Actors.Characters.Components;
+using Melia.Zone.World.Items;
+
+namespace Melia.Zone.Buffs.Handlers
```

Comment:
I agree that the "standard" status conditions can stay in the normal folder, though as far as I know this buff is tied to a specific skill.

Reply:
That's a matter of interpretation. Is a buff tied to a specific skill and class if, under normal circumstances, you only get it from that skill? What if they add an item that gives a certain buff, not necessarily this one, that was previously skill only? Do we move the file? What if a buff already belongs to multiple skills/items/whatever, indicated by not including a class name? What about the buffs that do include a class name, _because_ they're intended to be tied to a skill, do they go into that class' folder by default, no matter what? What if these buffs no longer belong to that class, because they were shuffled around with ReBuild? What about monster (skill) buffs, do we sort these into monster sub-folders?

These are rhetorical questions. I'm just trying to illustrate that giving order to the ~4600 buffs currently in the game is a bigger discussion than saying "sort them by what class you found them with first".